Monika-After-Story/MonikaModDev
# Add Monika ghost intro and Natsuki desk art

Natsuki desk art made by Iron-707

# TESTING
1. add `return store.mas_getEV("greeting_ghost")` to the first line in the `selectGreeting` function in `script-greetings`
2. Launch game and get scared

Unlocking:
- answering Yes to the spoops question will unlock the gret
- the greeting should lock itself
- there is an update script, but we can test that later.
